src/org/loklak/api/search/TimeAndDateService.java

```diff
@@ -62,8 +65,14 @@ public static SusiThought timeAndDate() {
 		JSONObject timeAndDate = new JSONObject();
 		
 		Date time_and_date = new Date();
-		timeAndDate.put("time_and_date", time_and_date.toString());
-		
+        String time_and_date_UTC = timeToUTC(time_and_date).toString().replaceAll(
+                time_and_date.toString().substring(20,23),
+                "UTC"
+        );
+
+        timeAndDate.put("local_time_and_date", time_and_date.toString());
```

Comment:
@vibhcool: Make your IDE use the `.editorconfig` file introduced recently. You will not face such errors.